<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>91</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>Entrez le titre : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="title_tf"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="maximumSize">
          <size>
           <width>91</width>
           <height>16</height>
          </size>
         </property>
         <property name="text">
          <string>Ajouter un fichier :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="filename_tf"/>
       </item>
       <item>
        <widget class="QPushButton" name="filename_btn">
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QComboBox" name="file_cb">
       <item>
        <property name="text">
         <string>default</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="list_element">
       <property name="editTriggers">
        <set>QAbstractItemView::CurrentChanged|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="elementStatus_cb">
         <item>
          <property name="text">
           <string>Ajouter (+)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Modifier (~)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Supprimer (-)</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="element_tf"/>
       </item>
       <item>
        <widget class="QPushButton" name="send_btn">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Valider</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionEnregistrer_au_format_TXT"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionEnregistrer_au_format_TXT">
   <property name="text">
    <string>Enregistrer au format TXT</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
